Client/Clique/Common/Constants/Constants.swift

```diff
@@ -6,13 +6,37 @@
 //
 
 import Foundation
+import Alamofire
+import UIKit
 
 enum Constants {
     
     enum Service {
         private static let baseURL = "https://cliquewebservice20221115180920.azurewebsites.net/api/"
         static let registerURL = baseURL.appending("Authentication/RegisterUser")
-        static let loginURL = baseURL.appending("Authentication/LoginUser")    
+        static let loginURL = baseURL.appending("Authentication/LoginUser")
+        static let profileGetUserURL = baseURL.appending("User")
+        static let profileUpdateURL = baseURL.appending("User/UpdateUserData")
+        static let passwordUpdateURL = baseURL.appending("User/UpdateUserPassword")
+        
+        static let headers: HTTPHeaders = [
+            "Authorization": "Bearer"
+            + " " + "\(UserStorage.token!)",
+            "Accept": "application/json"
+        ]
```

Comment:
you should never force unwrap UserStorage.token! Also formatting:
Look about enums with associated values
```suggestion
        static let headers: HTTPHeaders = [
            "Authorization": "Bearer"
            + " " + "\(UserStorage.token!)",
            "Accept": "application/json"
        ]
```
```suggestion
        static let headers: HTTPHeaders = [
            "Authorization": "Bearer \(UserStorage.token!)",
            "Accept": "application/json"
        ]
```